requirements.txt

```diff
@@ -17,7 +17,7 @@ pydantic>=1.8.1,<2.0.0 # Required by fastapi
 PyJWT==2.4.0
 pyyaml>=5,<6
 requests>=2,<3
-sqlalchemy[asyncio]==1.4.36
+sqlalchemy[asyncio]==1.4.27
```

Comment:
@SteveDMurphy I'm assuming this bump down was required for the bigquery connector?

Reply:
Correct! My bad forgot to notate that 🙁